source/develop/dev-process/working-with-gerrit.md

```diff
@@ -42,17 +43,20 @@ Alternatively, sign in with:
 * Any fedora FAS account, using: &lt;username&gt;.id.fedoraproject.org
 
 
-Once you log in, choose a Gerrit username. This username will be used to generate an HTTP password (Settings --> HTTP Password).
+Once you log in, choose a Gerrit username. This username will be used to generate an HTTP password (Settings --> HTTP Password) and also needed to SSH connection.
 
+To choose username:
+* Go to Settings
+* In the username field, enter your username and click Save Changes
```

Comment:
Add a blank line below this line.

Reply:
if you intend to change this then it can be improved a bit with "Enter your username in the username field and click Save Changes"